### PR TITLE
Add guides catalog pages

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-15328

To enable the routes and links use the `quickstartsDebug` debug function.

### CODE REVIEW

**Review by fhlavac — APPROVED** [review]

File: src/js/App/Routes/QuickstartsCatalogRoute.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  const bundle = getUrl('bundle');
+  const disabled = useSelector(
+    ({
+      chrome: {
+        quickstarts: { disabled },
+      },
+    }) => disabled
+  );
+
+  useEffect(() => {}, [bundle]);
```
Comment (karelhala) [inline src/js/App/Routes/QuickstartsCatalogRoute.js:16]: Is this empty useEffect good for anything?

**Comment by fhlavac** [comment]
Tested locally, it seems to be working :-)

File: src/js/App/Routes/QuickstartsCatalogRoute.js (hunk 1 above)
Comment (Hyperkid123) [reply src/js/App/Routes/QuickstartsCatalogRoute.js:16]: Yes, there will be some additional logic once the backend is deployed.

Comment (karelhala) [reply src/js/App/Routes/QuickstartsCatalogRoute.js:16]: Can we add the logic once the backend is deployed? :smile:

Comment (Hyperkid123) [reply src/js/App/Routes/QuickstartsCatalogRoute.js]: done

**Pull request merged**